### ISSUE TITLE
The *Log in to flag review* button is hard to read

### ISSUE THREAD

**Issue by kumar303** [mozilla/addons-frontend#6656]
### Describe the problem and steps to reproduce it:

* Make sure you are signed out
* Go to any add-on
* Go to the reviews page
* For any review, click *Flag*

### What happened?

The button is hard to read

<img width="1372" alt="screenshot 2018-10-16 11 45 43" src="https://user-images.githubusercontent.com/55398/47032978-470a5280-d139-11e8-80d5-018ed4dc517d.png">

### What did you expect to happen?

The text should be white

### Anything else we should know?

This is a recent regression

**Comment by ioanarusiczki** [comment]
Verified on AMO dev and AMO stage with FF62 (Win10 and Android 8.0) -this issue no longer reproducible, the text is white. 

![not reproducible flag issue](https://user-images.githubusercontent.com/33448286/47083998-f703bc80-d21a-11e8-8811-491a52f98faa.png)
